querybook/server/lib/export/exporters/gspread_exporter.py

```diff
@@ -105,17 +106,20 @@ def export_form(self):
             ),
         )
 
-    def _get_max_rows(self, statement_execution_id, start_cell: str = "A1"):
-        result_columns_len = self._get_statement_execution_columns_len(
-            statement_execution_id
+    @with_session
+    def _get_max_rows(
+        self, statement_execution_id, start_cell: str = "A1", session=None
+    ):
+        result_columns_len = self._get_statement_execution_num_cols(
+            statement_execution_id, session=session,
         )
         start_cell_coord = worksheet_coord_to_coord(start_cell)
         sheet_start_coord = worksheet_coord_to_coord("A1")
 
         column_offset = start_cell_coord[0] - sheet_start_coord[0]
         row_offset = start_cell_coord[1] - sheet_start_coord[1]
 
-        total_column_cells = result_columns_len + column_offset
+        total_column_cells = max(result_columns_len + column_offset, 26)
```

Comment:
add a comment about 26, or move 26 to some constant like DEFAULT_GSPREAD_NUM_COLS